GoodThingsComein3s/ViewControllers/DetailsViewController.m

```diff
@@ -0,0 +1,144 @@
+//
+//  DetailsViewController.m
+//  GoodThingsComein3s
+//
+//  Created by Ruhee Rajwani on 7/28/22.
+//
+
+#import "DetailsViewController.h"
+#import "APIManager.h"
+#import "AFNetworking.h"
+#import "RestaurantPhotoCollectionViewCell.h"
+
+@interface DetailsViewController () <UICollectionViewDelegate,UICollectionViewDataSource, UICollectionViewDelegateFlowLayout>
+
+@property (weak, nonatomic) IBOutlet UILabel *restaurantNameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *restaurantPriceLabel;
+@property (weak, nonatomic) IBOutlet UILabel *restaurantCategoriesLabel;
+@property (weak, nonatomic) IBOutlet UILabel *restaurantAddressLabel;
+@property (weak, nonatomic) IBOutlet UICollectionView *restaurantPhotosCollectionView;
+@property (weak, nonatomic) IBOutlet UIImageView *restaurantRatingImageView;
+@property (nonatomic) NSDictionary *restaurantToShow;
```

Comment:
This would be a good place to reuse the `Restaurant` model class. Then instead of messing with dictionary syntax, you could access the restaurant's property